docs/src/man/comparisons.md

```diff
@@ -40,7 +40,7 @@ as row indices rather than a separate `id` column.
 | Column slicing by location | `df.iloc[:, 1:]`       | `df[:, 2:end]`                     |
 | Row indexing by label      | `df.loc['c']`          | `df[findfirst(==('c'), df.id), :]` |
 | Column indexing by label   | `df.loc[:, 'x']`       | `df[:, :x]`                        |
-| Column slicing by label    | `df.loc[:, ['x','z']]` | `df[:, [:x, :z]]`                  |
+| Column slicing by label    | `df.loc[:, ['x', 'z']]` | `df[:, [:x, :z]]`                  |
```

Comment:
can you please double check that if we loose alignment of `|` in markdown is not required in tables (I think it is not required).

Reply:
Almost certain that it's not required, but I will fix it anyway. No sense fixing one style thing and breaking another :laughing:

Reply:
great - and now you see why your work is that much appreciated 😄.

Reply:
In fixing this, I noticed an inconsistency with table formatting (specifically the alignment rows) in this file. Roughly half of them have spaces after and before the bars, while the others don't. That is

```
| :------ | ------- |
vs
|:--------|---------|
```

Would you like me to fix these up as well? I prefer the later style.

Reply:
I also prefer the latter. It would be great if you fixed these.